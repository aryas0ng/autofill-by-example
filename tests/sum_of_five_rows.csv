11,36
15,27
2,16
5,57
3,57
2,
4,
43,
5,
6,
9,
3,
4,
2,
5,
2,
6,
1,
8,
6,
1,
21,
3,