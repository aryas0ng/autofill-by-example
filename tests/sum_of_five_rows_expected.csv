11,36
15,27
2,16
5,57
3,57
2,60
4,67
43,66
5,27
6,24
9,23
3,16
4,19
2,16
5,22
2,23
6,22
1,37
8,39
6,31
1,25
21,24
3,3